tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -342,10 +342,12 @@ public function testListTableColumnsDispatchEvent()
 
         $this->_sm->dropAndCreateTable($table);
 
+        /** @var \PHPUnit_Framework_MockObject_MockObject $listenerMock */
```

Comment:
This shouldn't be needed.

Reply:
It still doesn't look right. PhpStorm unambiguously detects the `$listenerMockType` as `PHPUnit\Framework\MockObject\MockObject` based on the `@return` annotation of `PHPUnit\Framework\MockObject\MockBuilder::getMock()`. So should do PHPStan. Instead of annotating every call like this, we need to find what the underlying problem is and fix it.